configs/Default.yml

```diff
@@ -0,0 +1,118 @@
+Omega:
```

Comment:
Just noticing...Not sure why github thinks there are two new versions of this file. When I check out the branch, there is only one. No action needed, just a curiousity...